docs/howtos/ckeditor.md

```diff
@@ -2,7 +2,125 @@
+
+Here are a few common configurations. For more, [see the `sanitize-html` documentation](https://npmjs.org/package/sanitize-html).
+
+### Allow additional HTML tags
+
+The **default** `allowedTags` configuration from `sanitize-html` is:
+
+```javascript
+  allowedTags: [
+    'h3', 'h4', 'h5', 'h6', 'blockquote',
+    'p', 'a', 'ul', 'ol', 'nl', 'li',
+    'b', 'i', 'strong', 'em', 'strike', 'abbr',
+    'code', 'hr', 'br', 'div', 'caption',
+    'table', 'thead', 'tbody', 'tr', 'th', 'td', 'pre', 'iframe'
+  ],
+```
+
+To add tags to that list, you would include all of those tags you want to keep, then add the new ones.  Here is an example adding `sup` and `sub`:
+
+```javascript
+// lib/modules/apostrophe-rich-text-widgets/index.js
+
+module.exports = {
+  // The standard list copied from the module, plus sup and sub
+  sanitizeHtml: {
+    allowedTags: [
+      'h3', 'h4', 'h5', 'h6', 'blockquote',
+      'p', 'a', 'ul', 'ol', 'nl', 'li',
+      'b', 'i', 'strong', 'em', 'strike', 'abbr',
+      'code', 'hr', 'br', 'div', 'caption',
+      'table', 'thead', 'tbody', 'tr', 'th', 'td', 'pre', 'iframe',
+      'sup', 'sub' // ⬅ The new tags
+    ],
+    // ...,
+  }
+};
+```
+
+### Add text styles using classes
+
+You will often want to add styles to the rich text editor widget that use classes for visual styling. For example, you may have these configurations in an area with a rich text widget for a basic paragraph and two special styles:
+
+```django
+  {{ apos.area(data.page, 'body', {
+    widgets: {
+      'apostrophe-rich-text': {
+        toolbar: [ 'Styles', 'Bold', 'Italic', 'Link', 'Unlink' ],
+        styles: [
+          { element: 'p', name: 'Paragraph' },
+          {
+            element: 'p',
+            name: 'Featured text',
+            attributes: { class: 'featured-text' }
+          },
+          {
+            element: 'h3',
+            name: 'Section title',
+            attributes: { class: 'section-heading' }
+          }
+        ]
+      }
+    }
+  }) }}
+```
+
+You can allow these classes in two ways. First, you could allow all classes on those elements using `allowedAttributes`:
+
+```javascript
+// lib/modules/apostrophe-rich-text-widgets/index.js
+
+module.exports = {
+  // The standard list copied from the module, plus sup and sub
+  sanitizeHtml: {
+    // allowedTags: [...],
+    allowedAttributes: {
+      'p': ['class'],
+      'h3': ['class'],
+      // Include the default setting as well, or else links will break
+      'a': [ 'href', 'name', 'target' ]
+    },
+    // ...,
+  }
+};
+```
+
+You can open this up to allow the `class` attribute on any element by replacing those individual tag name keys in `allowedAttributes` with an asterisk string (`'*': ['class']`).
```

Comment:
Is there another way to call out this important bit? A little 'Did you Know'-type bar?